libs/tup-components/src/tickets/TicketCreateModal/TicketCreateModal.tsx

```diff
@@ -6,14 +6,21 @@ import styles from './TicketCreateModal.module.css';
 
 const TicketCreateModal: React.FC<
   React.PropsWithChildren<{
-    display: 'secondary' | 'link';
+    display: 'secondary' | 'secondary-small' | 'link';
   }>
 > = ({ children, display }) => {
   const [isOpen, setIsOpen] = useState(false);
   const toggle = () => {
     setIsOpen(!isOpen);
   };
 
+  const buttonAttrs:
+    | { type: 'secondary'; size?: 'small' | undefined }
+    | { type: 'link'; size: undefined } =
+    display === 'secondary-small'
+      ? { type: 'secondary', size: 'small' }
+      : { type: 'link', size: undefined };
```

Comment:
To make this less arcane, I can use the verbose (but clearer) syntax I reduced via https://github.com/TACC/tup-ui/pull/320/commits/3ea214c.

Reply:
Outdated. Deleted in ba14b22.